researchstack-sdk/build.gradle

```diff
@@ -22,7 +22,7 @@ android {
     versionName "1.0"
     multiDexEnabled true
     // hack until subpopulation Guid is properly handled and not hardcoded
-    buildConfigField 'String', 'STUDY_SUBPOPULATION_GUID', studyId // default subpopulation
+    buildConfigField 'String', 'STUDY_SUBPOPULATION_GUID', "\"studyId\"" // default subpopulation
```

Comment:
This should remain configurable by consuming packages, usually it's set through gradle.properties